tests/endpoints/test_schemas.py

```diff
@@ -163,23 +156,23 @@ class SchemaExample(schemas.Model):
 
     assert m.import_data(
         {"around": {"origin": "40.730610,-73.935242", "offset": "1km", "scale": "2km", "decay": "0.1"}}
-    ).to_primitive() == {"around": {"origin": u"40.730610,-73.935242", "decay": 0.1, "scale": u"2km", "offset": u"1km"}}
+    ).to_primitive() == {"around": {"origin": "40.730610,-73.935242", "decay": 0.1, "scale": "2km", "offset": "1km"}}
 
     with pytest.raises(schemas.SchematicsDataError):
         m.import_data({"around": "40.730610,-73.935242"}, validate=True)
 
 
-def test_area_model():
+def test_location_model():
     class SchemaExample(schemas.Model):
 
-        area = schemas.StringModelType(schemas.Area)
+        location = schemas.StringModelType(schemas.Location)
```

Comment:
Looks like it got things a bit odd because you just switched the order? Might be better to keep the previous to make sure that you didn't lose anything in the change?

Reply:
yep it was due to me removing the area class will bring back the previous area test cases.